WordPress/Classes/ViewRelated/NUX/LoginWPComViewController.swift

```diff
@@ -1,6 +1,8 @@
 import UIKit
 
 class LoginWPComViewController: SigninWPComViewController, LoginViewController {
```

Comment:
It also means more work later.  Its not really a lot of code to look at and i don't think it will be that difficult to review.  Doing it now will also avoid having to do more work (an extra round of review and testing) later.